### DISCUSSION TITLE
青龙面板定时规则  无法设置 每一个半小时执行一次脚本

### DISCUSSION THREAD

**Discussion by qq1768624192** [whyour/qinglong#1664] [Q&A]
青龙面板定时规则 无法设置 每一个半小时(或每一小时40分…………)执行一次脚本，
https://www.cnblogs.com/rainights/p/12573197.html
通过上方博客得知 多行cron表达式子才可以实现 该功能。
但是青龙面板只可以写一行cron表达式，而且无法使用 && 、|| 等符号 。
各位大佬，青龙现版本有实现该定时规则的方法么？？真心求解。

**Comment by xmoxmo** [comment]
必须半小时吗？这个是29分钟的：0-58/29 * * * *

**Reply by ayurnero** [reply]
他说的是1.5h
